stripe/api_resources/error_object.py

```diff
@@ -1,8 +1,26 @@
+from typing import Optional
+from stripe.api_resources.payment_intent import PaymentIntent
+from stripe.api_resources.setup_intent import SetupIntent
+from stripe.api_resources.source import Source
 from stripe.util import merge_dicts
 from stripe.stripe_object import StripeObject
 
+from api_resources.payment_method import PaymentMethod
```

Comment:
This import looks different from `PaymentIntent`, etc. above?

Reply:
Oh, good catch. Autoimport